packages/suins/sources/suins.move

```diff
@@ -224,16 +253,21 @@ public fun init_for_testing(ctx: &mut TxContext): SuiNS {
     };
 
     authorize_app<Test>(&admin_cap, &mut suins);
-    add_config(
-        &admin_cap,
-        &mut suins,
-        config::new(
-            b"000000000000000000000000000000000",
-            1200 * suins::constants::mist_per_sui(),
-            200 * suins::constants::mist_per_sui(),
-            50 * suins::constants::mist_per_sui(),
-        ),
+    let range1 = new_range(vector[3, 3]);
```

Comment:
I'd ideally keep the old config too, so the previous config tests also work for now.